packages/ifpack2/src/Ifpack2_BlockTriDiContainer_decl.hpp

```diff
@@ -0,0 +1,362 @@
+/*@HEADER
+// ***********************************************************************
+//
+//       Ifpack2: Tempated Object-Oriented Algebraic Preconditioner Package
```

Comment:
"Tempated" -> "Templated" (I fixed this typo in most Ifpack2 classes a while back)